### ISSUE TITLE
Arguments based on #[cfg(feature = "foo")] are not handled correctly

### ISSUE THREAD

**Issue by clesmian** [intellij-rust/intellij-rust#10222]
## Environment

* **IntelliJ Rust plugin version:** 0.4.189.5234-223
* **Rust toolchain version:** 1.67.1 (d5a82bbd2 2023-02-07) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2022.3.2 (CL-223.8617.54)
* **Operating system:** Linux 5.15.0-60-generic
* **Macro expansion:** enabled

## Problem description
Even though this code compiles without any issues with both feature `foo` enabled or disabled, CLion/Rust plugin complains about the number of parameters supplied to the function.
![screenshot](https://user-images.githubusercontent.com/18008727/224012978-f0ce727c-6142-41f0-9bab-4433ea70a20e.png)

In addition this one issue is reported as 2 Errors in the upper right corner. One is located in line 11 (at the function name) the other in line 16 (with the "third" argument). In the Problems view at the bottom there is only one error shown in line 11.

## Steps to reproduce
<details>
   <summary>Code</summary>

   ### main.rs
   ```rust
   fn requires_2_args(arg1: bool, arg2: i8) -> i8 {
       if arg1 {
           -arg2
       } else {
           arg2
       }
   }

   pub fn main() {
       println!("{}",
                requires_2_args(
                    #[cfg(not(feature = "foo"))]
                        false,
                    #[cfg(feature = "foo")]
                        true,
                    42
                )
       );
   }
   ```

### Cargo.toml
```toml
[package]
name = "rust-project"
edition = "2018"
version = "0.1.0"

[features]
foo = []
```

</details>
